src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_pix2pix_zero.py

```diff
@@ -974,6 +974,9 @@ def __call__(
         prompt_embeds_edit[1:2] += edit_direction
 
         # 10. Second denoising loop to generate the edited image.
+        self.scheduler.set_timesteps(num_inference_steps, device=device)
```

Comment:
@patrickvonplaten 

pix2pix_zero has two denoising loops so we have to reset the timesteps before the second one to set the `_step_index` to `None` and restart the counter again 

is this ok?